Proxy/faraday_uart_stack/layer_2_protocol.py

```diff
@@ -483,16 +484,18 @@ def GET(self):
 
     def run(self):
         while self.enable_flag:
-            time.sleep(0.001)
+            time.sleep(0.01)
             if not self.receiver_class.rx_packet_queue.empty():
-                data = self.receiver_class.rx_packet_queue.get()
-                try:
-                    unpacked_datalink = self.datalink_packet_struct.unpack(data)
-                    #Place datalink payload into payload queue
-                    self.rx_data_payload_queue.put(unpacked_datalink[3])
-                except:
-                    print "FAIL"
-                    pass  #print "Failed parsing" !!!!!FIX!!!!!
+                #  Loop through all known queue data and receive
+                for i in range(0, self.receiver_class.rx_packet_queue.qsize()):
+                    data = self.receiver_class.rx_packet_queue.get()
+                    try:
+                        unpacked_datalink = self.datalink_packet_struct.unpack(data)
+                        #Place datalink payload into payload queue
+                        self.rx_data_payload_queue.put(unpacked_datalink[3])
+                    except:
+                        print "FAIL"
```

Comment:
Don't mean to be picky but lets just update this to use logging and remove line 495 inline commented out print statement too.